index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>OS X Calculator</title>
+</head>
+<meta charset='utf-8' />
+  <link rel='stylesheet' href='style.css'/>
+<body>
+
+  <div class='calculator'>
+    <div id='screen'> 0
+    </div>
+    <div class='button'> AC
+    </div>
+    <div class='button'> +/-
```

Comment:
don't forget to namespace your css classnames so they stay unique `.button` should be `.calculator-button`